### DISCUSSION TITLE
Transfer weights from monai's hovernet to original author's hovernet

### DISCUSSION THREAD

**Discussion by rajeshtims** [Project-MONAI/MONAI#6651] [Q&A] [answered]
If I use monailabel to train Hovernet, how can I copy these weights to the original author's model (state dict mapping)?
The state dictionary mapping is quite strange here; names are completely different and not sure which key in monai's hovernet corresponds to which key in the original author's model.

**Comment by KumoLiu** [comment]
Hi @Rajeshtims, I think you could refer to these mapping rules.
https://github.com/Project-MONAI/MONAI/blob/0ffe55a0d2d9165dc3841e5cc4a9d177b08c6b57/monai/networks/nets/hovernet.py#L623-L650

Hope it can help you, thanks!

**Reply by rajeshtims** [reply]
Thanks for the quick reply. I am a bit confused about the mapping of the decoder. Could you please provide me some information about how to get the decoder mapping?

**Reply by KumoLiu** [reply, marked as answer]
Hi @Rajeshtims, we don't have the decoder mapping, maybe you need to manually export the structure of the network and compare them.
Thanks!
